Dockerfile

```diff
@@ -24,7 +26,7 @@ COPY --chown=carma . /home/carma/src/
 RUN ~/src/docker/checkout.bash
 RUN ~/src/docker/install.sh
 
-FROM usdotfhwastoldev/autoware.ai:develop
+FROM base_image 
```

Comment:
Is there difference between using base_image and setup here?

Reply:
So setup is a build stage built from the base_image.